beancount_reds_importers/importers/vanguard/tests/vanguard_test.py

```diff
@@ -7,13 +7,13 @@
     vanguard.Importer(
         {
             "account_number": "444555",
-            "main_account": "Assets:Vanguard:401k:{ticker}",
+            "main_account": "Assets:Vanguard:401k:{ticker}:{401ksrc}",
```

Comment:
Interesting, I didn't realize the fully meaning of "commodity-leaf". I prefer the order I have here because it yields "401k:VTIAX:Aftertax", "401k:VTIAX:Match", etc. all under VTIAX rather than "401k:Aftertax:VTIAX", "401k:Match:VTIAX", etc. But I'll just do this in my personal configuration.

Reply:
This also prompted me to rename the replacement field from '401ksrc' to 'source401k' so that it can be used as a Python identifier. Starting with a number was a problem. I also figure "source" is better than "src"